packages/on-chain/test/airdrop.behavior.spec.ts

```diff
@@ -0,0 +1,220 @@
+import path from "path";
+import { promises } from "fs";
+
+import { expect } from "chai";
+import { loadFixture, mine } from "@nomicfoundation/hardhat-network-helpers";
+import { ethers, run } from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { Contract, ContractReceipt, Transaction } from "ethers";
+import { StandardMerkleTree } from "@openzeppelin/merkle-tree";
+
+import template from "../tasks/defaultTemplate.json";
+import realityEthAbi from "../../../node_modules/@reality.eth/contracts/abi/solc-0.8.6/RealityETH-3.0.abi.json";
+import { getMerkleTree } from "../utils/merkleTree";
+
+describe("Airdrop", function () {
+  let deployer: SignerWithAddress,
+    user1: SignerWithAddress,
+    user2: SignerWithAddress;
+  let testAvatar: Contract,
+    realityModule: Contract,
+    shrine: Contract,
+    mock: Contract,
+    testToken: Contract,
+    realityEth: Contract,
+    templateId: string;
+
+  let tree: StandardMerkleTree<string[]>;
+
+  const timeout = 42;
+  const cooldown = 23;
+  const expiration = 0;
+  const bond = "1000";
+
+  const user1Amount = "50";
+  const user2Amount = "50";
+
+  async function deployFixture() {
```

Comment:
Not sure if fixtures are needed in other test files as well, but if so, it might be smart to move it to a separate file. In that way it can be added to and each test file only uses the object members they need from the return values

Reply:
+1

Reply:
This is the only test file.